.travis.yml

```diff
@@ -1,35 +1,44 @@
-language: cpp
+language: python
 
-os: 
-    - osx
-    - linux 
-
-env:
-    matrix:
-        - TEST_SUITE="bin/py.test -n 2 -vvs --ignore=tests/scancode --ignore=tests/extractcode --ignore=tests/licensedcode --ignore=tests/cluecode --ignore=tests/packagedcode"
-        - TEST_SUITE="bin/py.test -n 2 -vvs tests/scancode"
-        - TEST_SUITE="bin/py.test -n 2 -vvs tests/extractcode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/licensedcode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/cluecode"
-        - TEST_SUITE="bin/py.test -n 2 -s tests/packagedcode"
-        - TEST_SUITE="./etc/release/release.sh"
+python:
+  - 2.7.12
+  - 2.7.13
 
+matrix:
+  include:
+    - os: osx
+      language: generic
+      python:
+      osx_image: xcode7.3
+    - os: osx
+      language: generic
+      python:
+      osx_image: xcode8.2
 
 install:
-    - ./configure
+  - ./configure
+  - source bin/activate
+  - pip install pytest-cov coveralls
 
 script:
-    - $TEST_SUITE
+  - pytest -n 2 -vvs --cov=src --ignore=tests/scancode --ignore=tests/extractcode --ignore=tests/licensedcode --ignore=tests/cluecode --ignore=tests/packagedcode
+  - pytest -n 2 -vvs --cov=src/scancode     tests/scancode
+  - pytest -n 2 -vvs --cov=src/extractcode  tests/extractcode
+  - pytest -n 2 -s   --cov=src/licensedcode tests/licensedcode
+  - pytest -n 2 -s   --cov=src/cluecode     tests/cluecode
+  - pytest -n 2 -s   --cov=src/packagedcode tests/packagedcode
+  - ./etc/release/release.sh
+
+after_success:
+  - coveralls
 
 notifications:
-    irc:
-        channels:
-          - "chat.freenode.net#aboutcode"
-    on_success: change
-    on_failure: always
-    use_notice: true
-    skip_join: true
-    template:
-        - "%{repository_slug}#%{build_number} (%{branch}-%{commit}:%{author})-%{message}- %{build_url}"
-        - 
-        - 
+  irc:
+    channels:
+      - "chat.freenode.net#aboutcode"
+  on_success: change
+  on_failure: always
+  use_notice: true
+  skip_join: true
+  template:
+    - "%{repository_slug}#%{build_number} (%{branch}-%{commit}:%{author})-%{message}- %{build_url}"
```

Comment:
BTW, even if not part of this change, is the space before `%{build_url}` intended here?